src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/LiveMetricsTests.cs

```diff
@@ -45,7 +45,7 @@ public Task TestDefaultMetrics()
                 async (appRunner, apiClient) =>
                 {
                     using ResponseStreamHolder holder = await apiClient.CaptureMetricsAsync(await appRunner.ProcessIdTask,
-                        durationSeconds: 2);
```

Comment:
Done - I used an int since that's what the existing Live Metrics tests were already doing (as opposed to TimeSpan), but if we'd rather it be consistent I can change those tests to use TimeSpan instead.